runtime/criusupport/criusupport.cpp

```diff
@@ -513,7 +516,33 @@ Java_org_eclipse_openj9_criu_CRIUSupport_checkpointJVMImpl(JNIEnv *env,
 		}
 		Trc_CRIU_before_checkpoint(currentThread, checkpointNanoTimeMonotonic, checkpointNanoUTCTime);
 
+		syslogOptions = (char *)j9mem_allocate_memory(STRING_BUFFER_SIZE, J9MEM_CATEGORY_VM);
```

Comment:
can we put this stuff inside the internal JVM hooks?

Reply:
The change in question closes the system logger handle according to `-Xsyslog` setting, if we move it into an internal hook, dump messages triggered by tracepoints located after `runInternalJVMCheckpointHooks()` won't be logged which is not desirable.
In addition, `syslogOptions` values have to be saved into a global structure for `setLogOptions()` after restore. 
I think current code location is appropriate.